README.md

```diff
@@ -0,0 +1,90 @@
+# Kokkos Documentation Workflow <br/><br/>
+
+Here, we summarize the steps for setting up the kokkos-core-wiki repo, and a
+branch-based workflow for updating documentation in the main Kokkos project.  The kokkos-core-wiki is the
+canonical source of Kokkos documentation "Truth".  It is no longer possible to
+directly edit the main Kokkos project Wiki.<br/>
+
+Sandia employees: Creation of wholly new pages will require Review and Approval (R&A).  Check with your Sandia team lead, if you have questions about the R&A process.<br/><br/>
+
+* **Step 1) Create your own fork of the kokkos-core-wiki repo by navigating to `https://github.com/kokkos/kokkos-core-wiki`, and clicking "Fork" in the upper right of the github page**.<br/>
```

Comment:
I wouldn't have the steps in boldface when the code blocks are not.

Reply:
ok, cool.  I can fix this.